benchmarks/src/main/java/com/yahoo/omid/benchmarks/utils/TimeValueTimeUnit.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2011-2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.omid.benchmarks.utils;
+
+import java.util.concurrent.TimeUnit;
+
+// TODO: Maybe this should be moved to the commons module as a general utility
+public class TimeValueTimeUnit {
```

Comment:
I suggest using seconds or minutes. Yes, I'd prefer configure one parameter rather than two, make is less error prone. If you still prefer to have two for humans - let it be, but then convert back to seconds in the config class, so that it's it's nice long